README.adoc

```diff
@@ -7,11 +7,40 @@ image:https://img.shields.io/badge/Angular%20Version-4-green.svg[]
 image:https://badges.frapsoft.com/typescript/code/typescript.svg?v=101[]
 image:https://circleci.com/gh/aerogear/cordova-showcase-template.svg?&style=shield[link=https://circleci.com/gh/aerogear/cordova-showcase-template]
 
+[width="85%"]
+|===========================================================================================================
+|                 | *Project Info*                                                                            
+| License:        | Apache License, Version 2.0                                                             
+| Build:          | NPM                                                                                     
+| End User Documentation:  | link:https://docs.aerogear.org[]                                                 
+| Community Documentation: | link:https://www.aerogear.org[]                                                      
+| Issue tracker:  | link:https://issues.jboss.org/browse/AEROGEAR[] 
+| Mailing lists:  | link:https://groups.google.com/forum/#!forum/aerogear[aerogear-dev] 
+|===========================================================================================================
+
 A template application to showcase features from link:https://github.com/aerogear/aerogear-js-sdk[Aerogear Js SDK].
 
 As this application showcases features from link:https://github.com/aerogear/aerogear-js-sdk[Aerogear Js SDK] ensure you checkout the desired git tag for whatever SDK version you intend to run. Check link:https://www.npmjs.com/org/aerogear[NPM] for associated versions and releases.
+
+== Features
+* link:https://docs.aerogear.org/aerogear/latest/identity-management.html[AeroGear Identity Management Service]
+* link:https://docs.aerogear.org/aerogear/latest/device-security.html[Device Security Checks]
+* link:https://docs.aerogear.org/aerogear/latest/mobile-metrics.html[Mobile Metrics]
+* link:https://docs.aerogear.org/aerogear/latest/push-notifications.html[Push Notifications]
+
+== Pre-requisites
+To run this application, the following is required:
+
+* link:https://nodejs.org/en/[Node] - version 8.9.4
```

Comment:
I think should we also mention that the mobile SDKs are required as well, depending on the platform the developer want to build for.